testplans/graphsync/memory-stress-k8s.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+name = "memory-stress"
+
+[global]
+plan = "graphsync"
+case = "stress"
```

Comment:
Not sure how different this test plan is to the one that is at https://github.com/filecoin-project/lotus/tree/master/testplans/graphsync , but if they are different (and I believe they are), we should change the name of the `case` to something else.

The automated dashboards, such as https://ci.testground.ipfs.team/dashboard?task_id=c09omhl5p7a858f1470g rely on uniqueness of the name... i.e. if we have 2 testplans that do different stuff and share the same plan:case , then dashboards would be meaningless.

Nothing urgent, just explaining in case we decide we want to run this on TaaS as well, periodically, which I think would be nice.